libs/ui/src/lib/form/password-input/password-input.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { ErrorStateMatcher } from '@angular/material/core';
+import { boolean } from '@blockframes/utils/decorators/decorators';
+
+@Component({
+  selector: 'password-input',
```

Comment:
```suggestion
  selector: '[control] password-input',
```
Makes the control input mandatory